dist/tools/headerguards/check.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+# Copyright 2017 Oliver Hahm <oliver.hahm@inria.fr>
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+# customizable
+DIRS="core drivers pkg sys"
```

Comment:
True. I saw some warnings and thought they were vendor specific, but actually they can and should be fixed. Will update.